multimedia/libvpx/files/patch-build-make-configure.sh.diff

```diff
@@ -1,11 +1,29 @@
---- build/make/configure.sh.orig	2020-08-01 00:27:21.000000000 +0200
-+++ build/make/configure.sh	2020-08-01 00:29:56.000000000 +0200
-@@ -854,7 +854,7 @@
-       fi
-       ;;
+--- build/make/configure.sh.orig	2020-11-26 14:23:30.000000000 -0800
++++ build/make/configure.sh	2020-11-26 14:26:37.000000000 -0800
+@@ -845,16 +845,8 @@
+   # Handle darwin variants. Newer SDKs allow targeting older
+   # platforms, so use the newest one available.
+   case ${toolchain} in
+-    arm*-darwin*)
```

Comment:
Your patch removes the `arm*-darwin*` case because it was iOS specific, but there's still an `x86*-darwin*` case below. Are we sure that whatever the `x86*-darwin*` case is doing for x86 macOS isn't also needed for arm64 macOS?

Reply:
The `x86*-darwin*` version was setting `-isysroot` to the MacOSX SDK, which wasn't needed for my build, but indeed looks important. I've fixed this now.

Reply:
BTW, I just noticed [#59360](https://trac.macports.org/ticket/59360), which now is triggered on arm by including the `x86*-darwin*` case.

This entire case block with the nonsense `osx_sdk_dir="$(configure.sdkroot)"` line has no effect apart from emitting a non-fatal `command not found` error.

So the correct `-isysroot` cflags and `-syslibroot` ldflags were actually being set on my system already without this block--though I'm not sure exactly how. It looks like we should just remove this block entirely.